src/sharing/components/Recipient.jsx

```diff
@@ -10,24 +10,28 @@ import { getDisplayName, getPrimaryEmail, getPrimaryCozy } from '..'
 
 const MAX_DISPLAYED_RECIPIENTS = 3
 
-export const RecipientsAvatars = ({ recipients, link, size }) => (
-  <div className={styles['recipients-avatars']}>
-    {link && <AvatarLink size={size} />}
-    {recipients.length > MAX_DISPLAYED_RECIPIENTS && (
-      <AvatarPlusX
-        extraRecipients={recipients
-          .slice(MAX_DISPLAYED_RECIPIENTS)
-          .map(recipient => getDisplayName(recipient))}
-        size={size}
-      />
-    )}
-    {recipients
-      .slice(0, MAX_DISPLAYED_RECIPIENTS)
-      .map(recipient => (
-        <Avatar name={getDisplayName(recipient)} size={size} />
-      ))}
-  </div>
-)
+export const RecipientsAvatars = ({ recipients, link, size, className }) => {
+  // we slice first to clone the original array because reverse() mutates it
```

Comment:
Aaargh good catch.
That being said, the comment mentioning `reveser()` only makes sense in the context of this PR. Maybe a better comment for the future would be to explain why we're reversing the array?

IIRC it's so that the recipients you've added last show up in the avatars.